GoogleDataTransport/GDTCORTests/Unit/GDTCORFlatFileStorageTest.m

```diff
@@ -66,351 +87,155 @@ - (void)tearDown {
                 });
   // Destroy these objects before the next test begins.
   self.testBackend = nil;
-  self.testPrioritizer = nil;
-  [[GDTCORRegistrar sharedInstance] reset];
-  [[GDTCORFlatFileStorage sharedInstance] reset];
   [GDTCORFlatFileStorage sharedInstance].uploadCoordinator =
       [GDTCORUploadCoordinator sharedInstance];
   self.uploaderFake = nil;
 }
 
+/** Generates and returns a set of events that are generated randomly and stored.
+ *
+ * @return A set of randomly generated and stored events.
+ */
+- (NSSet<GDTCOREvent *> *)generateEventsForStorageTesting {
+  GDTCORFlatFileStorage *storage = [GDTCORFlatFileStorage sharedInstance];
+  NSMutableSet<GDTCOREvent *> *generatedEvents = [[NSMutableSet alloc] init];
+  // Generate 100 test target events
+  for (int i = 0; i < 100; i++) {
+    GDTCOREvent *event = [GDTCOREventGenerator generateEventForTarget:kGDTCORTargetTest
+                                                              qosTier:nil
+                                                            mappingID:nil];
+    [generatedEvents addObject:event];
+    [storage storeEvent:event onComplete:nil];
+  }
+
+  // Generate 50 FLL target events.
+  for (int i = 0; i < 50; i++) {
+    GDTCOREvent *event = [GDTCOREventGenerator generateEventForTarget:kGDTCORTargetFLL
+                                                              qosTier:nil
+                                                            mappingID:nil];
+    [generatedEvents addObject:event];
+    [storage storeEvent:event onComplete:nil];
```

Comment:
Should we wait for completion here? I guess we don't have to since storage should enqueue operations and if we have to, then it may be an actual storage issue

Reply:
Wasn't necessary.